internal/pkg/middleware/cors.go

```diff
@@ -7,6 +7,12 @@ import (
 func EnableCORS(h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.Header().Set("Access-Control-Allow-Origin", "*")
-		h.ServeHTTP(w, r)
+		w.Header().Set("Access-Control-Allow-Methods", "POST, GET, OPTIONS, PUT, DELETE")
```

Comment:
If I were u, I will just do `handlers.CORS()` (see github.com/gorilla/handlers)

Reply:
package middleware


import (
	"net/http"

	"github.com/gorilla/handlers"
)

func EnableCORS(h http.Handler) http.Handler {
	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		handlers.CORS()
	})
}
I tried but it's didn't work